polkadot/runtime/rococo/build.rs

```diff
@@ -14,12 +14,21 @@
 // You should have received a copy of the GNU General Public License
 // along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
 
+use std::env;
 use substrate_wasm_builder::WasmBuilder;
 
+// note: needs to be synced with rococo-runtime-constants::time hard-coded string literal in
+// prod_or_fast macro.
+const ROCOCO_EPOCH_DURATION_ENV: &str = "ROCOCO_EPOCH_DURATION";
+
 fn main() {
-	WasmBuilder::new()
-		.with_current_project()
-		.import_memory()
-		.export_heap_base()
-		.build()
+	let mut builder = WasmBuilder::new().with_current_project().import_memory().export_heap_base();
+
+	if env::var(ROCOCO_EPOCH_DURATION_ENV).is_ok() {
+		builder = builder.enable_feature("fast-runtime")
+	};
+
+	builder.build();
+
+	println!("cargo:rerun-if-env-changed={}", ROCOCO_EPOCH_DURATION_ENV);
 }
```

Comment:
```suggestion
}

#[cfg(not(feature = "std"))]
fn main() {}
```

Reply:
Please also make `substrate-wasm-builder` an optional dep. (This is done to speedup the build process)

Reply:
done in d55ae688a8